src/widgets/FeedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,127 @@
+import { AxiosError } from 'axios'
```

Comment:
```suggestion
import type { AxiosError } from 'axios'
```

Если мы импортируем тип, то делаем это так, что дать понять вебпаку, что его не надо включать в бандл